src/components/CreatePost.tsx

```diff
@@ -0,0 +1,152 @@
+import { useForm } from 'react-hook-form';
+import React, { useState, useRef } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { CreatePoster } from '../apis/poster';
+import PrimaryButton from './common/PrimaryButton';
+import { useNavigate } from 'react-router-dom';
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  Text,
+  Stack,
+  Image,
+  Flex,
+  Spacer,
+  Textarea,
+  Center,
+} from '@chakra-ui/react';
+import {
+  ChevronLeftIcon,
+  PlusSquareIcon,
+  SmallCloseIcon,
+} from '@chakra-ui/icons';
+
+export interface NewPostData {
+  title: string;
+  image: FileImage[];
+  channelId: string;
+}
+
+export interface FileImage {
+  id: string;
+  file: File;
+}
```

Comment:
```suggestion
type FileImage = {
  id: string;
  file: File;
}

type NewPostData =  {
  title: string;
  image: FileImage[];
  channelId: string;
}
```

`interface`가 아닌 `type`으로 수정해주시고 상관은 없지만 문맥상 흐름을 위해 `FileImage`를 먼저 선언해주시면 좋을 것 같아요!!